source/fundamentals/authentication.txt

```diff
@@ -142,6 +143,180 @@ string as follow:
 
    To learn more on specifying the default mechanism, see :ref:`csharp-authentication-default`.
 
+.. _csharp-aws:
+
+MONGODB-AWS
+~~~~~~~~~~~
+
+.. note::
+
+   The ``MONGODB-AWS`` authentication mechanism is available for
+   MongoDB deployments on MongoDB Atlas.
+
+The ``MONGODB-AWS`` authentication mechanism uses your Amazon Web Services
+Identity and Access Management (AWS IAM) credentials to authenticate your
+user. To learn more about configuring MongoDB Atlas, see the
+:atlas:`Set Up Passwordless Authentication with AWS IAM Roles </security/passwordless-authentication/#set-up-passwordless-authentication-with-aws-iam-roles>` guide.
+
+To instruct the driver to use this authentication mechanism, you can specify
+``MONGODB-AWS`` either as a parameter in the connection string or by using
+the ``MongoCredential`` constructor and passing "MONGODB-AWS" as the authentication mechanism.
+
+Learn how to specify this authentication mechanism and the various ways to
+provide your AWS IAM credentials in the next sections.
+
+These sections contain code examples that use the following placeholders:
+
+* ``awsKeyId`` - value of your AWS access key ID
+* ``awsSecretKey`` - value of your AWS secret access key
+* ``atlasUri`` - hostname and port of your MongoDB Atlas deployment
+* ``awsSessionToken`` - value of your AWS session token
+
+Specify Your Credentials in the Environment
```

Comment:
Suggestion:
This specific wording, "the Environment", comes from the corresponding Java doc was selected due to the various ways, including the AWS SDK for Java, that the driver could provide the credentials -- not just through environment variables.

I think this section only describes environment variables, so it could be more precise to call this section "Specify Your Credentials in Environment Variables". If there's an AWS SDK for .NET integration that's coming soon, I think it's fine to leave as is.

Reply:
I reworded this because not all of the AWS SDK retrieval methods involve environment variables (namely, locally stored profiles).